frb_codegen/src/generator/dart/mod.rs

```diff
@@ -317,6 +317,9 @@ fn generate_dart_implementation_body(spec: &DartApiSpec, config: &Opts) -> Acc<D
         )
     }));
 
+    lines.push(section_header("wire2api"));
+    lines.push(dart_wire2api_funcs.join("\n\n"));
+
```

Comment:
nit: Looks like these lines are moved from below to here. If this is not a necessary change, maybe we should revert it? otherwise the generated code will be changed while not necessary

Reply:
Yeah, it is needed to include the wire2api into the class, so we can use the 'this'

Reply:
![image](https://user-images.githubusercontent.com/5236035/194676987-e5824b41-722a-42e1-860e-2faba5e95bb8.png)

I mean, shall we put it in line 333 as the old case? otherwise the generated output changes a lot while seems not necessary.

Reply:
No, as they require to be inside the class (Closed on line 323) to use the `this` class.